<launch>
  <node pkg="octocostmap" type="octocostmap" name="octomapper">
    <param name="map_frame" value="/openni_camera" />
    <param name="map_resolution" value="0.025" />
    <remap from="cloud" to="/camera/depth/points_passthrough" />
  </node>

</launch>
